app/views/storage/_storage_pod_folders.html.haml

```diff
@@ -5,7 +5,7 @@
       %tbody
         - @folders.each do |f|
           %tr{:title => _("View '%{name}' Cluster") % {:name => f.name},
-            :onclick => "miqTreeActivateNode('storage_pod_tree', 'dsc_xx-#{to_cid(f.id)}');"}
+            :onclick => "miqTreeActivateNode('storage_pod_tree', 'xx-#{to_cid(f.id)}');"}
```

Comment:
@skateman Since 10767 removed every `id_prefix` from every tree, can you check that there are no more bugs like this? ;)